### PR TITLE
Update the landscape contributing section for timeline changes

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
**Description**
Added steps to add a new service mesh to the timeline.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ The service mesh landscape is powered by Jekyll. In order to add/update the land
 
 Entries should be listed in alphabetical order. Edit other yaml files as needed.
 
+- Aslo, to update the Service Mesh Timeline, add the new service mesh in the [non-functional.yml](https://github.com/layer5io/layer5/blob/master/_data/categories/non-functional.yml) file. Add `announce-date` key for the service mesh then add a `weight` key having value 1+`maximum value of weight`, to list the service mesh in the timeline correctly.
```
Comment (leecalcote) [inline CONTRIBUTING.md:21]: Thank you. I don't know that the links in the list items above are still valid.

Comment (Nikhil-Ladha) [reply CONTRIBUTING.md:21]: Oops. They are not valid. Didn't check that earlier. Will update the links.

**Comment by nik-redhat** [comment]
Updated the links @leecalcote

**Comment by leecalcote** [comment]
@Nikhil-Ladha I wonder if we might change "weight" to something like "timeline-order"...

**Comment by Nikhil-Ladha** [comment]
Makes sense, will do it.

**Comment by Nikhil-Ladha** [comment]
Done!

**Review by leecalcote — APPROVED** [review]
Fantastic. :)

**Pull request merged**